code/game/atoms.dm

```diff
@@ -54,17 +54,21 @@
 			//we were deleted
 			return
 
-//Called after New if the map is being loaded. mapload = TRUE
-//Called from base of New if the map is not being loaded. mapload = FALSE
-//This base must be called or derivatives must set initialized to TRUE
-//must not sleep
-//Other parameters are passed from New (excluding loc), this does not happen if mapload is TRUE
-//Must return an Initialize hint. Defined in __DEFINES/subsystems.dm
-
-//Note: the following functions don't call the base for optimization and must copypasta:
-// /turf/Initialize
-// /turf/open/space/Initialize
-
+/**
+  * Called after New if the map is being loaded. mapload = TRUE
+  * Called from base of New if the map is not being loaded. mapload = FALSE
+  *
+  * This base must be called or derivatives must set initialized to TRUE
+  *
+  * Must not sleep
+  *
+  * Other parameters are passed from New (excluding loc), this does not happen if mapload is TRUE

```

Comment:
does anyone know what this means

Reply:
That's just weird way of saying when loading from map arguments will be default. (There's no place to take them from anyway)